experimental/packages/opentelemetry-instrumentation/test/node/EsmInstrumentation.test.mjs

```diff
@@ -26,14 +26,21 @@ class TestInstrumentationWrapFn extends InstrumentationBase {
     super('test-esm-instrumentation', '0.0.1', config);
   }
   init() {
-    console.log('test-esm-instrumentation initialized!');
     return new InstrumentationNodeModuleDefinition(
       'test-esm-module',
       ['*'],
       moduleExports => {
-        this._wrap(moduleExports, 'testFunction', () => {
-          return () => 'patched';
+        const wrapRetval = this._wrap(moduleExports, 'testFunction', () => {
```

Comment:
```suggestion
        const wrappedReturnValue = this._wrap(moduleExports, 'testFunction', () => {
```
This is a total nit, but I wonder if we could use a full name for this - primarily because this tends to be a confusing functionality and I'm a sucker for detailed variable names. Maybe something like `patchedModule`, similar to the `patchedRequest` you have in the http code? Nonblocking, especially since it's a test.